release-notes.md

```diff
@@ -4,6 +4,17 @@ Here are the release notes for the Experience Platform Mobile SDKs and Adobe Pla
 
 **Important:** Adobe Places is currently in Beta. On the Places GA date, the Places release notes content will be removed from this release notes and the location to the Places release notes will be provided.
 
+## November 15, 2019
+
+The following updates were made in this release:
+
+**iOS Identity 2.1.3**
+
+- Synced custom and advertising identifiers with nil or empty values are now cleared from Identity shared state and local storage. They are also not synced with the Experience Cloud ID (ECID) Service.
+- Fixed a threading issue where the Experience Cloud ID (ECID) Service response was handled on an incorrect thread potentially causing a memory corruption crash.
+
+These changes were released as part of ACPCore pod v2.3.6.
```

Comment:
I would change "pod" to "CocoaPod", or just remove it as there is the manual install option.